pyproject.toml

```diff
@@ -8,7 +8,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = ">=3.9,<3.13"
```

Comment:
I see in this latest commit that you removed support for 3.8 again. Is there a specific reason for this? If it is not supported anymore, it should be indicated in the changelog under "Removed".

Reply:
Would suggest setting version to `<=3.12` instead of `<3.13` to be more explicit, e.g. in case a version 3.125 is introduced (which although unlikely, is possible).

Reply:
I just responded on the above comment for this